torchvision/models/inception.py

```diff
@@ -393,24 +384,29 @@ def _forward(self, x):
         outputs = [branch1x1, branch3x3, branch3x3dbl, branch_pool]
         return outputs
 
-    def forward(self, x):
+    def forward(self, x: Tensor) -> Tensor:
         outputs = self._forward(x)
         return torch.cat(outputs, 1)
 
 
 class InceptionAux(nn.Module):
 
-    def __init__(self, in_channels, num_classes, conv_block=None):
+    def __init__(
+        self,
+        in_channels: int,
+        num_classes: int,
+        conv_block: Optional[Callable[..., nn.Module]] = None
+    ):
         super(InceptionAux, self).__init__()
         if conv_block is None:
             conv_block = BasicConv2d
         self.conv0 = conv_block(in_channels, 128, kernel_size=1)
         self.conv1 = conv_block(128, 768, kernel_size=5)
-        self.conv1.stddev = 0.01
+        setattr(self.conv1, 'stddev', 0.01)
```

Comment:
Did `mypy` complain about a non-existing attribute here? If yes, I would rather silence the `mypy` error instead of making this working code less legible.

Reply:
Precisely! I just pushed your suggestion